Note that not all Rancher CRDs are available through the API yet (#1043)
rancher/rancher-docs: docs/api/api-reference.mdx

title: API Reference
---

:::note

> At this time, not all Rancher resources have been made available through the Rancher Kubernetes API.

Reviewer: ```suggestion
At this time, not all Rancher resources are available through the Rancher Kubernetes API.
```

Just suggesting word choice update to avoid passive voice.

> At this time, not all Rancher resources have been made available through the Rancher Kubernetes API.

Reviewer: ```suggestion
At this time, not all Rancher resources are available through the Rancher Kubernetes API.
```

Just suggesting a word choice edit to avoid passive voice
